encuentro/ui/main.py

```diff
@@ -342,6 +340,8 @@ def show_message(self, err_type, text):
                     text = text.decode("latin1")
                 except UnicodeDecodeError:
                     text = repr(text)
+            except AttributeError:
+                pass
 
```

Comment:
Never hide an error like this. I prefer it to explode and we will debug the problem.